src/rgw/driver/rados/rgw_cr_rados.cc

```diff
@@ -738,7 +738,6 @@ int send_sync_notification(const DoutPrefixProvider* dpp,
       ldpp_dout(dpp, 1) << "ERROR: " << __func__
                         << ": caught buffer::error couldn't decode TagSet "
                         << dendl;
-      return -EIO;
```

Comment:
the return value from `send_sync_notification()` is ignored anyway.
so, it is porbably ok to return `-EIO` here, so we are consistent with the other places in this function where we return an error.

Reply:
done